tests/test_test_plugins.py

```diff
@@ -31,6 +31,13 @@ def test_plugin_name(self):
 		plugin_test_runner = PluginTestRunner(DUMMY_PLUGIN_PATH)
 		assert plugin_test_runner.plugin_name == DUMMY_PLUGIN
 
+	def test_get_conda_base(self):
+		plugin_test_runner = PluginTestRunner(DUMMY_PLUGIN_PATH)
+		try:
+			plugin_test_runner.get_conda_base()
+		except CalledProcessError as e:
+			assert False, e
```

Comment:
I added the assertion, but the test is meant to fail if this specific exception is raised (it indicates that the `conda` base cannot be accessed). If this isn't good testing practice, let me know.

Reply:
The test will fail if _any_ exception is raised though. Are other exceptions okay? I would have thought any exception is bad.

Reply:
My thought process was that we do want the test to fail if it encounters any exception (because exceptions are bad in general) but we might like to explicitly mark this particular exception. This is probably unnecessary & I've removed it in favor of your original suggestion.